intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorFeature.kt

```diff
@@ -16,8 +16,13 @@ data class KtorFeature(
   override val icons: List<PluginIcon> = emptyList(),
   override val pluginVersion: String? = null,
   override val pluginId: String? = null,
+  val ktorVersion: String? = null,
```

Comment:
what the difference between ktorVersion and pluginVersion? probably it should be the same

Reply:
No, `pluginVersion` is a version of the plugin (feature). And `ktorVersion` is a version of Ktor framework that this feature was compiled with.